var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -120,7 +120,6 @@ class Dealii(CMakePackage):
     def cmake_args(self):
         spec = self.spec
         options = []
-        options.extend(std_cmake_args)
 
         # CMAKE_BUILD_TYPE should be DebugRelease | Debug | Release
         for word in options[:]:
```

Comment:
you can also delete this filtering part because there won't be anything to filter now. You could keep the comment, but move closer to `-DCMAKE_BUILD_TYPE=` below.